### DISCUSSION TITLE
Question about tabsets with groups set not appearing across pages if some tabs don't have content

### DISCUSSION THREAD

**Discussion by nickvigilante** [quarto-dev/quarto-cli#10370] [Q&A] [answered]
### Description

I'm not sure if this is a bug, so I'm gonna start it off as a discussion.

We make use of tabsets across our docs, and some tabsets pertain to different programming languages. Not all tabsets share the same exact tabs (e.g., some tabsets can have Python and Golang, others can have Python and C++, others can have Java and R, etc.).

When we use the same group across tabsets where a previously selected tab is present in one tabset but missing in another tabset, we see a missing tab:

![image](https://github.com/user-attachments/assets/2b02973c-a300-4502-935e-18d257112d81)

![image](https://github.com/user-attachments/assets/3368720f-69e3-4161-9c09-9207669e19fe)

Note how when Golang is selected, the second code block displays nothing. I have this reproduced here:

https://github.com/nickvigilante/tabset-group-issue

Is this a bug, or is this expected? We can work around the problem, but I want to know sooner rather than later, so I can start the work. I guess it would be nice to see if we switch tabs that either the previously selected tab in the tabset stays (e.g., Python), or if navigating across pages, the first tab available displays.

**Comment by cscheid** [comment]
This is a bug.

**Reply by nickvigilante** [reply]
Cool, want me to log this? I'll write it up.

**Reply by cscheid** [reply, marked as answer]
I already opened, thanks! (https://github.com/quarto-dev/quarto-cli/issues/10371)
